legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/PureModel.java

```diff
@@ -141,6 +141,8 @@ public class PureModel implements IPureModel
     final MutableMap<String, Root_meta_pure_runtime_PackageableRuntime> packageableRuntimesIndex = Maps.mutable.empty();
     final MutableMap<String, Runtime> runtimesIndex = Maps.mutable.empty();
 
+    public static final PureModel SYSTEM_PURE_MODEL = getSystemPureModel();
```

Comment:
Views on this?
needed so that we can load contract without creating pureModel/ExecutionContext everytime. Might be useful at other places too

Reply:
Should we call it CORE_PURE_MODEL as its made up of all core repositories?